### PR TITLE
update metrics guide for new grafana dashboards

### PR DESCRIPTION
**Pull request by sebito91**
## What did you implement:

Simple updates to the metrics markdown docs.

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: contrib/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# Event Gateway Contributions 
+# Event Gateway Contributions
 
 This folder contains services and contributions which complement the Event Gateway.
 
-+ Install Event Gateway on Kubernetes using [helm](helm/README.md)
-+ Provision the Event Gateway on AWS ECS Fargate using [terraform](terraform/README.md)
+- Install Event Gateway on Kubernetes using [helm](helm/README.md)
```
Comment (mthenw) [inline contrib/README.md]: helm -> Helm

File: contrib/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+- Provision the Event Gateway on AWS ECS Fargate using [terraform](terraform/README.md)
```
Comment (mthenw) [inline contrib/README.md]: terraform -> Terraform

File: docs/metrics.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+   1. [Prometheus]#prometheus)
+   1. [InfluxDB](#influxdb)
+1. [List of Metrics](#list-of-metrics)
+   1. [Events API](#events-api)
+   1. [Configuration API](#configuration-api)
+
+### Introduction
+
+The Event Gateway exposes a number of [Prometheus](https://prometheus.io/)-based metrics + counters via the configuration
+API to help monitor the health of the gateway. The table below outlines the specific metrics available to query from the endpoint:
```
Comment (mthenw) [inline docs/metrics.md]: Which table? :)

File: docs/metrics.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+1. [Introduction](#introduction)
+1. [Collecting Metrics](#collecting-metrics)
+1. [Visualizing Data](#visualizing-data)
+   1. [Prometheus]#prometheus)
+   1. [InfluxDB](#influxdb)
+1. [List of Metrics](#list-of-metrics)
+   1. [Events API](#events-api)
+   1. [Configuration API](#configuration-api)
+
+### Introduction
```
Comment (mthenw) [inline docs/metrics.md]: I would move the intro paragraph above the `Contents` and remove the "Introduction" point from TOC.

File: docs/metrics.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+Importing a grafana [dashboard](http://docs.grafana.org/reference/export_import/#export-and-import) is relatively straightforward provided your datasources are configured
+properly within your grafana instance. Each of our dashboards lists respective datasources as a template variable, assuming the default datasource is set
+to the type for your datasource; in other words, if you import the Prometheus dashboard the template will expect your default data source to be set
+to Prometheus.
+
+If your default datasource is not either of the Prometheus or InfluxDB sources, fear not! The template variable is set as the datasource for each
+panel within the dashboard, so updating the datasource is as easy as choosing another option from the template dropdown.
+
+**NOTE:** There are two dashboards included for each datasource: an aggregated dashboard for all spaces in a given eventgateway deployment, and a second
```
Comment (mthenw) [inline docs/metrics.md]: eventgateway -> Event Gateway

File: docs/metrics.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+- [Splunk](https://splunkbase.splunk.com/app/4077/#/details)
+- and others ...
+
+### Visualizing Data
+
+Included in the [contrib](../contrib/grafana) folder are identical dashboards to visualize Event Gateway metrics with different
+data sources. Depending on your setup you'll find a [Prometheus](../contrib/grafana/prometheus) and [InfluxDB](../contrib/grafana/influxdb) version
+of the dashboard.
+
+Importing a grafana [dashboard](http://docs.grafana.org/reference/export_import/#export-and-import) is relatively straightforward provided your datasources are configured
```
Comment (mthenw) [inline docs/metrics.md]: grafana -> Grafana

File: docs/metrics.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+properly within your grafana instance. Each of our dashboards lists respective datasources as a template variable, assuming the default datasource is set
```
Comment (mthenw) [inline docs/metrics.md]: grafana -> Grafana

File: docs/metrics.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+   1. [Configuration API](#configuration-api)
+
+### Introduction
+
+The Event Gateway exposes a number of [Prometheus](https://prometheus.io/)-based metrics + counters via the configuration
+API to help monitor the health of the gateway. The table below outlines the specific metrics available to query from the endpoint:
+
+### Collecting Metrics
+
+Event Gateawy metrics are exposed via a Prometheus text-based metric [exporter](https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-example) and as such are
```
Comment (mthenw) [inline docs/metrics.md]: Gateawy -> Gateway

File: docs/metrics.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
+# Collecting and Viewing Event Gateway Metrics
+
+This guide details how to collect and analyze metrics from the Event Gateway.
+
+## Contents
+
+1. [Introduction](#introduction)
+1. [Collecting Metrics](#collecting-metrics)
+1. [Visualizing Data](#visualizing-data)
+   1. [Prometheus]#prometheus)
```
Comment (mthenw) [inline docs/metrics.md]: broken link

**Comment by sebito91** [comment]
Thanks for the feedback, will address shortly!

**Comment by mthenw** [comment]
@sebito91 Please also update the link in repo README https://github.com/serverless/event-gateway#reference

**Pull request merged**